docs/UserGuide.md

```diff
@@ -321,7 +321,7 @@ Format: `list [LIST_PARAMETER]`
 | Pet       | pets, pet, p, -p, /p             |
 
 Examples:
-* `list buyers`, lists all Buyer contacts.
+* `list buyer`, lists all Buyer contacts.
 * `list supplier`, lists all Supplier contacts.
 * `list d`, lists all Deliverer contacts.
 * `list-o`, lists all Orders.
```

Comment:
Can you ensure that we pass CI? I am a bit worried that the grading script does not recognise that we wrapped up v1.3 if we don't pass CI.

Reply:
Yup I'm doing that.